modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,276 @@
+[id="cluster-logging-collector-log-forward-cloudwatch_{context}"]
+= Forwarding logs to Amazon CloudWatch
+
+You can forward logs to Amazon CloudWatch, a monitoring and log storage service hosted by Amazon Web Services (AWS). You can forward logs to CloudWatch in addition to, or instead of, the default OpenShift Logging-managed Elasticsearch log store.
+
+To configure log forwarding to CloudWatch, create a `ClusterLogForwarder` custom resource (CR) with an output for CloudWatch and a pipeline that uses the output.
+
+.Procedure
+
+. Create a `Secret` YAML file that uses the `aws_access_key_id` and `aws_secret_access_key` fields to specify your base64-encoded AWS credentials. For example:
```

Comment:
Later on in the document, like in the sections, `Example: Replacing the default infrastructure name` and `Example: Naming log groups for application namespaces` you don't precede  text boxes with `For example` you simple precede it with a colon. 

I think you could do the same here. Cutting down on word count might make the commands slightly faster to go through. 

Or conversely, throughout the entire doc, you could precede text boxes with, `For example:` or `in the following:` or whatever introductory phrase. Or you could simply leave it as it is :)